packages/calcite-components/src/components/shell-center-row/shell-center-row.scss

```diff
@@ -36,19 +40,19 @@
   @apply self-start;
 }
 
-:host([height-scale="s"]) {
+:host .height-s {
```

Comment:
is `:host` needed here for specificity or could we remove it? Applies to similar lines.